stripper/ze_sky_athletic_adv_v9_12.cfg

```diff
@@ -0,0 +1,40 @@
+;Make tram before boss arena unblockable so CTs cant delay using it or glitch it so it doesnt despawn in RTV level.
+modify:
+{
+	match:
+	{
+		"classname" "func_movelinear"
+		"targetname" "lv14 train"
+	}
+	replace:
+	{
+		"blockdamage" "999999"
+	}
+}
+
+;Disable skybox level since it is incredibly laggy (probably) due to spamming a trigger on every player. Instead reset to warmup level instead of fixing the trigger's spam detection, since a box is not a fun level anyways.
```

Comment:
What exactly is "skybox level"? I've played through the map a fair number of times and never experienced this lag you mention.

Reply:
If you include all levels, skybox would be level 10 (warmup, 1-3, 1-3 ex, truth, rtv, then skybox). It is what occurs when you beat rtv level or lose it 3 times. It is just where players spawn in for every level, but removed the remote destination from the teleport around that spawn in area, so it keeps players trapped in just a skybox. I assume it is the trigger lagging the server, since it is still active from what I could tell, just doesn't have a destination set due to all levels' brushes being enabled, stopping the map from tping the players anywhere (it may be something else, since I didn't bother testing it to find out if it is really that, just seems like it might be caused by that when I quickly looked in hammer). It lags 100% of the time that we have 64 players in the area, especially after zombies spawn in and people start shooting them.